backend/shared/src/create-notification.ts

```diff
@@ -981,17 +982,16 @@ export const createUniqueBettorBonusNotification = async (
   // Only send on 5th bettor
   if (uniqueBettorsExcludingCreator.length !== TOTAL_NEW_BETTORS_TO_REPORT)
     return
-  const mostRecentUniqueBettors = await getValues<User>(
-    firestore
-      .collection('users')
-      .where(
-        'id',
-        'in',
-        uniqueBettorsExcludingCreator.slice(
-          uniqueBettorsExcludingCreator.length - TOTAL_NEW_BETTORS_TO_REPORT,
-          uniqueBettorsExcludingCreator.length
-        )
-      )
+
+  const lastBettorIds = uniqueBettorsExcludingCreator.slice(
+    uniqueBettorsExcludingCreator.length - TOTAL_NEW_BETTORS_TO_REPORT,
+    uniqueBettorsExcludingCreator.length
+  )
+
+  const mostRecentUniqueBettors = await pg.map(
+    `select * from users where id in $1`,
```

Comment:
does that syntax work? I thought it had to be of the type:
`id = ANY ($1)`

Reply:
https://www.postgresql.org/docs/current/functions-comparisons.html#FUNCTIONS-COMPARISONS-IN-SCALAR

`in` and `any` are similar. `any` is specifically for Postgres arrays and `in` is for lists of arbitrary expressions.